### ISSUE TITLE
Can you offer the pytorch XLNet pretrained model?

### ISSUE THREAD

**Issue by tyj117** [lonePatient/Bert-Multi-Label-Text-Classification#31]
Hi,

Thanks for your project. Now I am trying to use the XLNet to train the model.
Can you offer the pytorch XLNet pretrained model?
I can only find the tensorflow XLnet model.

Thanks,
YJ

**Comment by jcliu0428** [comment]
Hi,
I think you can find what you need here:
https://s3.amazonaws.com/models.huggingface.co/
